webview/src/plots/components/ZoomedInPlot.tsx

```diff
@@ -23,8 +29,31 @@ export const ZoomedInPlot: React.FC<ZoomedInPlotProps> = ({
     }
   }, [])
 
+  const onPlotChange = () => {
+    const actions = zoomedInPlotRef.current?.querySelector('.vega-actions')
+    const createdRawDataAction = actions?.querySelector(
+      `.${styles.vegaCustomAction as string}`
+    )
+    if (createdRawDataAction) {
+      return
+    }
+    const rawDataAction = document.createElement('a')
+    rawDataAction.textContent = 'Save as Raw Data'
+    rawDataAction.addEventListener('click', () => {
+      exportPlotAsRawData(id, props.data)
```

Comment:
From my understanding of how our plots work, custom plot data will always be in `props.data`, but I wasn't sure if template plots will always hold their data in`props.spec.data` (where our plots have it). I got the data for template plots from the "back-end" for now.